Gems/PhysX/Code/Editor/EditorJointConfiguration.cpp

```diff
@@ -512,4 +514,37 @@ namespace PhysX
                 followerEntityName.c_str());
         }
     }
+
+    void EditorJointMotorConfig::Reflect(AZ::ReflectContext* context)
+    {
+        if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<EditorJointMotorConfig>()
+                    ->Version(1)
+                    ->Field("Use Motor", &EditorJointMotorConfig::m_useMotor)
+                    ->Field("Force Limit", &EditorJointMotorConfig::m_driveForceLimit)
+                    ->Field("Gear Ratio", &EditorJointMotorConfig::m_gearRatio)
+                    ;
+
+            if (auto* editContext = serializeContext->GetEditContext())
+            {
+                editContext->Class<PhysX::EditorJointMotorConfig>(
+                                "PhysX Joint Motor Configuration", "")
+                        ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
+                        ->Attribute(AZ::Edit::Attributes::Category, "PhysX")
+                        ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
+                        ->DataElement(0, &PhysX::EditorJointMotorConfig::m_useMotor, "Use Motor"
+                                , "Enable motor in the joint")
+                        ->DataElement(0, &PhysX::EditorJointMotorConfig::m_gearRatio, "Gear Ratio"
+                                , "Sets gear ratio")
+                        ->DataElement(0, &PhysX::EditorJointMotorConfig::m_driveForceLimit, "Force Limit Value"
+                                , "Sets force limit value")
+                        ;
+            }
+        }
+    }
+
+    JointMotorProperties EditorJointMotorConfig::ToGameTimeConfig() const{
+        return JointMotorProperties(m_useMotor, m_gearRatio, m_driveForceLimit);
+    }
```

Comment:
nit/standards: Might be worth running clang-format over this code to make sure braces appear on a new line etc...

Reply:
This function was removed.